### PR TITLE
Replacing #[global] Instance with #[export] Instance

### PR DESCRIPTION
**Pull request by DafinaTrufas**

### CODE REVIEW

**Review by palmskog — APPROVED** [review]
Looks good.

File: scripts/replacing-global.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+BASE=$(dirname $0)
+find . -iname "*.v" -exec sed -i -e 's/#\[global\]\(.*\)Instance/#\[export\]\1Instance/' "{}" \;
```
Comment (wkolowski) [inline scripts/replacing-global.sh:3]: Newline missing at the end.

Also, this script doesn't catch newlines between `#[global]` and `Instance`, right? Did you hunt these cases down manually?

**Review by wkolowski — APPROVED** [review]

File: scripts/replacing-global.sh (hunk 1 above)
Comment (DafinaTrufas) [reply scripts/replacing-global.sh:3]: Oh, I found here https://www.gnu.org/software/sed/manual/html_node/Regular-Expressions.html, that ". Matches any character, including newline."

Comment (wkolowski) [reply scripts/replacing-global.sh:3]: The thing is, `sed` works line-by-line. So there's no newline to be matched.

Comment (wkolowski) [reply scripts/replacing-global.sh]: But I did some `grep`ping and it seems you didn't miss anything.

Comment (DafinaTrufas) [reply scripts/replacing-global.sh]: Ah, ok, I see. From what I've seen, there are no cases like the one you mentioned. Would it be sufficient to manually double check that there aren't, or should I find a way to include this check in the script?

Comment (wkolowski) [reply scripts/replacing-global.sh]: No need to bother. `grep -h "\#\[global\]" $(find -name "*.v")` says it's all `#[global] Hint Mode...`, so you can go on and merge the PR.

**Review by wkolowski — CHANGES_REQUESTED** [review]
It seems you got some kind of regression, as the `strip_newlines` script got deleted.

**Comment by DafinaTrufas** [comment]
I deleted `strip_newlines.py`, because I had this script actually duplicate (`strip_newlines.py` and `strip-newlines.py`). I know you've also told me to remove one of the two, and I did it, but for some reasons today there were again both there, so I had to fix the problem.

**Review by wkolowski — APPROVED** [review]

**Pull request merged**